build/packer/README.md

```diff
@@ -1,34 +1,32 @@
 # build/packer
 
-This directory contains [Packer] templates that automate building VM images.
+This directory contains Packer templates that automate building VM images.
 Each `.json` file is a packer template.
 
 ## To use
 
-1. Install Packer:
-    ```bash
-    brew install packer
-    ```
-2. Configure `gcloud` with your personal [User Application Default Credentials][gauth].
-3. Run:
-   ```bash
-   packer build <VM_TEMPLATE>.json
-   ```
-
-The location of the created VM image will be printed when the build completes.
+1. Find the build configuration you need in TeamCity under `Internal > Cockroach > Build > Infrastructure`
+2. Trigger a run on the right branch
+
+## To rotate GCP credentials
+1. Authenticate using your `-a` account and generate a new key.
+```
+gcloud iam service-accounts keys create new_packer_key.json \
+    --iam-account=packer@crl-teamcity-agents.iam.gserviceaccount.com
+```
```

Comment:
nit: If you indent these code blocks by 4 spaces, it will show indented in the rendered text. Also, it'll make the raw markdown easier to read.